Explorer/Assets/Scripts/CommunicationData/URLHelpers/URNExtensions.cs

```diff
@@ -0,0 +1,19 @@
+namespace CommunicationData.URLHelpers
+{
+    public static class URNExtensions
+    {
+        // TODO: would be ideal that urns are type specific
```

Comment:
You can introduce a named structure to achieve it, though it's optional of course